.ebextensions/cwl-webrequest-metrics.config

```diff
@@ -0,0 +1,193 @@
+###############################################################################
+## Option Settings
+##    Namespace: "aws:elasticbeanstalk:application:environment"
+##    OptionName: WebRequestCWLogGroup
+##       Default:   <EnvironmentName>-webrequests
+##    Description: This is the name of the cloudwatch log group for web requests (access log)
+##
+## To get an SNS alert, add a subscription to the Elastic Beanstalk Notification
+##   topic.  (e.g. set your Notificiation Endpoint to your email address)
+##
+## Metrics
+##  Namespace:  ElasticBeanstalk/<EnvironmentName>
+##  Metrics:    CWLHttp4xx  CWLHttp5xx
+##
+## Cloudwatch Alarms
+##   CWLHttp5xxCount   - this alarm fires if the number of calls returning
+##                        5xx response codes > 10
+##   CWLHttp4xxPercent - this alarm fires if the percentage of calls returning
+##                        4xx response codes > 10%
+##
+###############################################################################
+
+# Nginx access log pattern:
+# $remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for"
+Mappings:
+  CWLogs:
+    WebRequestLogGroup:
+      LogFile: "/var/log/nginx/access.log"
+      TimestampFormat: "%d/%b/%Y:%H:%M:%S %z"
+    FilterPatterns:
+      Http4xxMetricFilter: "[..., status=4*, size, referer, agent, xforward]"
+      HttpNon4xxMetricFilter: "[..., status!=4*, size, referer, agent, xforward]"
+      Http5xxMetricFilter: "[..., status=5*, size, referer, agent, xforward]"
+      HttpNon5xxMetricFilter: "[..., status!=5*, size, referer, agent, xforward]"
+    RailsProductionLogGroup:
```

Comment:
For all custom logs, `Mappings`, `Outputs` and `Resources` need to be created. This tells to read the Rails production logs from `/var/log/eb-docker/containers/eb-current-app/production.log` and specifies the timestamp format in the logs. I didn't modify the timestamp format because I recall that it looks the same as in the other log files on Papertrail.
We'd add these sections for the other files we want to tail as well. What I'm unclear about is the file formatting for `FilterPatterns`. In other words, what log group do these filter patterns apply to if the indentation needs to be on this level? They should apply to `WebRequestLogGroup` but I find this confusing.